src/components/NcRichContenteditable/NcAutoCompleteResult.vue

```diff
@@ -32,10 +32,10 @@
 			</div>
 		</div>
 
-		<!-- Label and subline -->
+		<!-- Title and subline -->
 		<span class="autocomplete-result__content">
-			<span class="autocomplete-result__label">
-				{{ label }}
+			<span class="autocomplete-result__title">
```

Comment:
```suggestion
			<span class="autocomplete-result__title" :title="title">
```
The only thing I could suggest to justify this change, and to provide ability to see long names fully:
![image](https://github.com/nextcloud/nextcloud-vue/assets/93392545/f6c17c06-8742-4254-843e-b9b656c3eb65)